src/cappa/base.py

```diff
@@ -186,23 +183,34 @@ def wrapper(_decorated_cls):
 
 
 def collect(
-    obj: type,
+    obj: type[T] | Command[T],
     *,
     backend: typing.Callable | None = None,
     version: str | Arg | None = None,
     help: bool | Arg = True,
     completion: bool | Arg = True,
-    color: bool = True,
-):
-    if not color:
-        # XXX: This should probably be doing something with the Output rather than
-        #      mutating global state.
-        os.environ["NO_COLOR"] = "1"
+) -> Command[T]:
+    """Retrieve the `Command` object from a cappa-capable source class.
 
+    Arguments:
+        obj: A class which can represent a CLI command chain.
+        backend: A function used to perform the underlying parsing and return a raw
+            parsed state. This defaults to constructing built-in function using argparse.
+        version: If a string is supplied, adds a -v/--version flag which returns the
+            given string as the version. If an `Arg` is supplied, uses the `name`/`short`/`long`/`help`
+            fields to add a corresponding version argument. Note the `name` is assumed to **be**
+            the CLI's version, e.x. `Arg('1.2.3', help="Prints the version")`.
+        help: If `True` (default to True), adds a -h/--help flag. If an `Arg` is supplied,
+            uses the `short`/`long`/`help` fields to add a corresponding help argument.
+        completion: Enables completion when using the cappa `backend` option. If `True`
+            (default to True), adds a --completion flag. An `Arg` can be supplied to customize
+            the argument's behavior.
+        color: Whether to output in color.
```

Comment:
It looks like you removed the parameter and forgot to remove its corresponding item in the docstring?